nemo_curator/scripts/download_and_extract.py

```diff
@@ -116,6 +117,13 @@ def attach_args(
         required=False,
         help="Path to input data directory",
     )
+    parser.add_argument(
+        "--input-meta",
```

Comment:
You should add this argument to every script in this folder that deals with a single dataset as input.

Reply:
I tried to do it to any script that invoked (directly or indirectly) any of the methods where `input_meta` is used. Please, let me know if I have missed anything. Thanks!